core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProvider.scala

```diff
@@ -158,28 +159,25 @@ class CompactEntityProvider(requestArguments: EntityRequestArguments,
                               parentContext: RawlsRequestContext
   ): Future[Int] =
```

Comment:
I did not change the signature of `deleteEntities` to accept `EntityKey` instead of `AttributeEntityReference` even though that would be semantically correct; I didn't want to cause fallout to `LocalEntityProvider` and any of its test code. I'm open to making this change if you think it's worth it; it wouldn't be too bad.

Reply:
I'd be in favor of it -- if we're making this change, might as well go all the way IMO

Reply:
changed!